packages/react-native/src/Authenticator/Defaults/ResetPassword/types.ts

```diff
@@ -1,10 +1,16 @@
-import { StyleProp, ViewStyle } from 'react-native';
+import { TextStyle, ViewStyle } from 'react-native';
 
 import { AuthenticatorComponentDefaults } from '@aws-amplify/ui-react-core';
 import { FormHandlers } from '../../types';
 
 export interface ResetPasswordStyle {
```

Comment:
Just a heads up, this file is gone now. This interface can be moved in to the component file

Reply:
The sub components have a similar structure so it would be beneficial to have common styles we can extend from.